basxconnect/core/views/menu_views.py

```diff
@@ -49,3 +52,33 @@
         settingsgroup,
     )
 )
+
+menu.registeritem(
+    menu.Item(
+        Link(
+            reverse("bread.views.system.systeminformation"),
+            _("System Information"),
+        ),
+        admingroup,
+    )
+)
+
+menu.registeritem(
+    menu.Item(
+        Link(
+            reverse("basxconnect.core.views.settings_views.maintenancesettings"),
+            _("Maintenance"),
+        ),
+        admingroup,
+    )
+)
+
+menu.registeritem(
+    menu.Item(
+        Link(
+            reverse("core.vocabulary.browse"),
+            _(basxconnect.core.models.Vocabulary._meta.verbose_name_plural),
```

Comment:
The strings inside ```Model._meta.verbose_name``` and ```Model._meta.verbose_name_plural``` should normally be already translated, so no need to use ```gettext``` or ```_()``` here.